lib/sprockets/sassc_processor.rb

```diff
@@ -0,0 +1,63 @@
+require 'sprockets/sass_processor'
+require 'base64'
+
+module Sprockets
+  class SasscProcessor < SassProcessor
+    def initialize(options = {}, &block)
+      @cache_version = options[:cache_version]
+      @cache_key = "#{self.class.name}:#{VERSION}:#{Autoload::SassC::VERSION}:#{@cache_version}".freeze
+      @importer_class = options[:importer]
+      @functions = Module.new do
+        include SassProcessor::Functions
+        include options[:functions] if options[:functions]
+        class_eval(&block) if block_given?
+      end
+    end
+
+    def call(input)
+      context = input[:environment].context_class.new(input)
+
+      options = engine_options(input, context)
+      engine = Autoload::SassC::Engine.new(input[:data], options)
```

Comment:
I believe it does have caching, but there’s currently no option to turn it on or off.